fastify.js

```diff
@@ -358,6 +360,16 @@ function build (options) {
     return this._contentTypeParser.hasParser(contentType)
   }
 
+  function handleClientError (e, socket) {
+    socket.end(
+      JSON.stringify({
+        error: http.STATUS_CODES['400'],
+        message: 'Client Error',
+        statusCode: 400
+      })
```

Comment:
According to documentation, the socket should be ended with a valid http response.
Here, the socket will be closed with a json output.
Maybe a simple "HTTP/1.1 400 Bad Request\r\n\r\n" can be used